packages/browser/src/lib/sdk.ts

```diff
@@ -0,0 +1,57 @@
+import { createAndBind } from '@sentry/core';
+import { BrowserOptions } from './backend';
+import { BrowserFrontend } from './frontend';
+
+export { addBreadcrumb, captureEvent, setUserContext } from '@sentry/core';
```

Comment:
Double `@sentry/core` import (line 1 and line 5), unless there's a need for a proper imports order (which is a code-smell anyway imo)

Reply:
One is import and one a re-export, not sure if you or I missed something 😅

Reply:
Silly me 🤦‍♂️